filereaderjs.gemspec

```diff
@@ -18,6 +18,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.11"
+  spec.add_development_dependency 'bundler', '~> 2.2.18'
```

Comment:
Why the change in quotes?

Reply:
This was copied from https://github.com/bonusly/tangolicious/pull/1